xbmc/filesystem/CurlFile.h

```diff
@@ -209,5 +211,6 @@ namespace XFILE
       MAPHTTPHEADERS m_requestheaders;
 
       long            m_httpresponse;
+      bool            m_isshoutcast;
```

Comment:
but it makes perfect sense to let consumers of this fileitem know that tag reading will fail.
Would renaming this to something generic like "canhavetags" be better?